### PR TITLE
Some Refactoring and Addition of Configuration

### PR DESCRIPTION
**Pull request by Zorro909**

### CODE REVIEW

File: src/main/resources/RiftSystem.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+allowedWorlds:
+ - "daily-world"
+ - "daily-world.*"
+
+portal_spawn_probability: 0.005
```
Comment (SticksDev) [inline src/main/resources/RiftSystem.yml:5]: In YML, there should be a new line to tell YML that is the EOF (End Of File)

**Review by SticksDev — CHANGES_REQUESTED** [review]
Great pull, code looks solid, you need to change some things, before we merge.

- Add 1 new line at the end of `src/main/resources/RiftSystem.yml ` Because YML requires one new line to call EOF (End Of File), or an error will be thrown when parsing.
- `.gitingore`  your `.settings` direcotry for eclipse. You can see a guide [here](https://linuxize.com/post/gitignore-ignoring-files-in-git/)

**Comment by Zorro909** [comment]
The yml works as it is, so I don't know why it should throw an error, but I can change that later.

Thanks for reminding me to add eclipse files to the .gitignore :)

And well, it's still Work In Progress, so there's still gonna be some things that will change before merging.

File: src/main/resources/RiftSystem.yml (hunk 1 above)
Comment (Zorro909) [reply src/main/resources/RiftSystem.yml:5]: As far as I know, EOF is just standard EOF and there's no need to add a Line Break there...
I mean I did already test it and it's working completely fine.

Furthermore the YAML Specs don't require that at all anyway, so I'd be interested in where you got that from?

**Comment by SticksDev** [comment]
Huh, okay. Github was complaining

**Comment by pyxld-kris** [comment]
This is sooo much prettier, thank you! I also REALLY appreciate the addition of 'allowedWorlds' in the config. That's going to let us ship this way sooner 🥇

**Review by pyxld-kris — APPROVED** [review]
🔥

**Pull request merged**